src/hooks/useForm.js

```diff
@@ -26,6 +27,7 @@ const reducer = (state, action) => {
       return {
         ...state,
         status: action.payload,
+        isLoading: action.payload === SUBMITTING ? true : false,
```

Comment:
do not expose extra boolean. Use the `formStatus` only

Reply:
Fixed.